### ISSUE TITLE
[Bug]: Since version 1.4.3, changing the value of a widget will not trigger the callback function defined by node.addWidget

### ISSUE THREAD

**Issue by lldacing** [Comfy-Org/ComfyUI_frontend#1627]
### Frontend Version

>=1.4.3

### Expected Behavior

1. Environment Preparation
The test node belongs to [Davemane42/ComfyUI_Dave_CustomNode](https://github.com/Davemane42/ComfyUI_Dave_CustomNode)
By the way, installing this custom node requires changing the way static resources are loaded in __init__.py, because I don't know where to start, Comfyui will not load resources copied to the directory `web/extensions`. Some custom_nodes directly copy static resources to this directory instead of specifying it through `WEB_DIRECTORY`.
![image](https://github.com/user-attachments/assets/5268299c-fa46-43e7-874a-023db5feb77f)

2. Expected Behavior

![1](https://github.com/user-attachments/assets/f1d22377-eb4e-4a64-847e-91687bfc24a7)

### Actual Behavior

In fact, the callback function of the widget is not triggered
![2](https://github.com/user-attachments/assets/dd8a4307-cfff-431b-a920-6605abed3c39)

### Steps to Reproduce

See Expected Behavior

### Debug Logs

```powershell
none
```

### Browser Logs

none

### Setting JSON

[comfy.settings.json](https://github.com/user-attachments/files/17843718/comfy.settings.json)

### What browsers do you use to access the UI ?

Microsoft Edge

**Comment by mijuku233** [comment]
If you enter the value directly, does it work correctly?

**Comment by lldacing** [comment]
> If you enter the value directly, does it work correctly?

yes

**Comment by mijuku233** [comment]
Try to update to the latest version of the front end. A similar bug was previously reported in #1603, and the latest front end fixes it. If the bug still exists after you update to the latest front end, then this may be a different but similar new bug.

**Comment by lldacing** [comment]
> Try to update to the latest version of the front end. A similar bug was previously reported in #1603, and the latest front end fixes it. If the bug still exists after you update to the latest front end, then this may be a different but similar new bug.

I have try 1.4.3-1.4.6

**Comment by mijuku233** [comment]
@webfiltered It seems that the previous bug affects more than just the combo widget

**Comment by webfiltered** [comment]
@lldacing Can you confirm that this _does_ work in 1.4.2, please?

**Comment by lldacing** [comment]
> @lldacing Can you confirm that this _does_ work in 1.4.2, please?

Sure, the first gif is in 1.4.2

**Comment by mijuku233** [comment]
> @lldacing Can you confirm that this _does_ work in 1.4.2, please?

I first encountered this bug in version 1.3.31. At that time, the bug only affected the nodes I wrote myself. Starting from version 1.4.3, it affects both vhs and ComfyUI_Dave_CustomNode.

**Comment by webfiltered** [comment]
VHS is fixed now, though, right?  I'm not sure it's the same issue, is all - might just be causing the same results.

**Comment by mijuku233** [comment]
> VHS is fixed now, though, right? I'm not sure it's the same issue, is all - might just be causing the same results.

vhs behaves normally in the latest version, but this may be a long-standing bug that has just been discovered.

**Comment by mijuku233** [comment]
@webfiltered 
1.4.2
![bug142](https://github.com/user-attachments/assets/78a88223-e1aa-4ecf-895e-250d60bb86f1)
1.4.7
![bug147](https://github.com/user-attachments/assets/ff433db0-9258-49fb-89a5-4edaa2334574)


I reproduced the bug I found two weeks ago, which is the opposite of #1603.
In version 1.4.2 (or earlier), clicking left and right to adjust the value of the widget can trigger the onWidgetChanged callback, while directly entering the value or selecting the value from the drop-down list cannot trigger the callback.
After version 1.4.3, the onWidgetChanged callback cannot be triggered at all.

            const onWidgetChanged = nodeType.prototype.onWidgetChanged;
            nodeType.prototype.onWidgetChanged = function (widget, value) {
                onWidgetChanged?.apply(this, arguments);
                this.widgets[0].hidden = this.widgets[1].value !== "images";
                this.widgets[0].value = 'frame: ' + this.widgets[2].value;
            };
